packages/api/orm/migration/1638312144983-initial.ts

```diff
@@ -0,0 +1,15 @@
+import * as fs from "fs";
+import { join } from "path";
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class initial1638312144983 implements MigrationInterface {
+  private readonly migrationId = "1638312144983-initial";
```

Comment:
This is fine for now but I am guessing that we can use reflection to avoid having to copy the name here next time.